### ISSUE TITLE
how to decide best model

### ISSUE THREAD

**Issue by blessxu** [gjy3035/C-3-Framework#35]
Since we use test set as validation, how to decide the model of best result. Just use the best result from every epoch? Or we train a fixed epoch and use the result of final epoch?  Thanks!

**Comment by gjy3035** [comment]
If you want to post the results in your paper, please split the training set into two parts for training and validation. During the training process, you can select the best model (best MAE or best val loss) on validation and then evaluate the model on the test set. 

Since everyone may have a different strategy to select the validation data (5%, 10%, etc.), we do not provide our validation data. At the same time, we use the test set for validation to guarantee that everyone can reproduce each experiment successfully.

**Comment by blessxu** [comment]
Thanks!
